src/extension.ts

```diff
@@ -105,13 +105,17 @@ function getClientConfig(context: ExtensionContext) {
     ['diagnostics.onParse', 'diagnostics.onParse'],
     ['diagnostics.onType', 'diagnostics.onType'],
     ['codeLens.localVariables', 'codeLens.onLocalVariables'],
+    ['formatting.enabled', 'formatting.enabled'],
   ];
   let clientConfig = {
     launchCommand: '',
     cacheDirectory: '',
     workspaceSymbol: {
       sort: false,
-    }
+    },
```

Comment:
This should not be needed, the `['formatting.enabled', 'formatting.enabled']` bit should handle injecting it into `clientConfig`

Reply:
Done.